ras/ras_ppcutils.py

```diff
@@ -14,8 +14,10 @@
 # Copyright: 2016 IBM
 # Author: Pavithra <pavrampu@linux.vnet.ibm.com>
 # Author: Sachin Sant <sachinp@linux.ibm.com>
+# Author: Shirisha Ganta <shirisha.ganta1@ibm.com>
```

Comment:
A common practice is to use vnet id for Author and also for sign-off,  ignore if the id is vnet id

Reply:
from next Patch onwards will do that as currently I need to do few more steps to get the vnet id..